File: spring-boot-java/src/main/java/io/ssaragi/java/spring/boot/dto/response/exam/ExamListResModel.java
package io.ssaragi.java.spring.boot.dto.response.exam;

import io.ssaragi.java.spring.boot.dto.data.exam.ExamData;
import io.ssaragi.java.spring.boot.dto.response.BaseApiResponseModel;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.util.List;

@Getter
@Setter
@ToString
@EqualsAndHashCode(callSuper = false)
public class ExamListResModel extends BaseApiResponseModel {

    private List<ExamData> list;
}

File: spring-boot-kotlin/src/main/kotlin/io/ssaragi/kotlin/spring/boot/dto/response/exam/ExamListResModel.kt
package io.ssaragi.kotlin.spring.boot.dto.response.exam

import io.ssaragi.kotlin.spring.boot.dto.data.exam.ExamData
import io.ssaragi.kotlin.spring.boot.dto.response.BaseApiResponseModel

data class ExamListResModel(

    var list: List<ExamData>? = null,

) : BaseApiResponseModel()
